internal/cli/iam/projects/create.go

```diff
@@ -74,51 +76,90 @@ func (opts *CreateOpts) newRegionUsageRestrictions() string {
 }
 
 func (opts *CreateOpts) newCreateProjectOptions() *atlas.CreateProjectOptions {
-	return &atlas.CreateProjectOptions{ProjectOwnerID: opts.projectOwnerID}
+	return &atlas.CreateProjectOptions{
+		ProjectOwnerID: opts.projectOwnerID}
```

Comment:
[super-nit] either rollback this change or put the closing brackets in a new line, this version of the code confuses me 
```suggestion
	return &atlas.CreateProjectOptions{
		ProjectOwnerID: opts.projectOwnerID,
	}
```